lighthouse-extension/app/src/lightrider-entry.js

```diff
@@ -9,6 +9,7 @@ const lighthouse = require('../../../lighthouse-core/index');
 
 const assetSaver = require('../../../lighthouse-core/lib/asset-saver.js');
 const LHError = require('../../../lighthouse-core/lib/lh-error.js');
+const preprocessor = require('../../../lighthouse-core/lib/proto-preprocessor.js');
```

Comment:
kinda wish we had a diff name/verb for this. 

proto-prepper?
iunno.

Also we could use an ascii diagram somewhere:

```

lighthouse_result.proto -> make_proto.py -> lighthouse_result.pb2
                                                            ⭏ 
lhr.json --> Proto processor -> lhr.processed.json -> make_roundtrip.py -> lhr.round_trip.json
```

Reply:
Added to the README in proto.